hdeps/cli.py

```diff
@@ -176,7 +177,11 @@ def main(
 
     def solve() -> None:
         for dep in deps:
-            walker.feed(Requirement(dep))
+            if dep.startswith(".") or "/" in dep:
```

Comment:
This is reasonable for a convenience. The day you can support git urls as well, you could do this instead:
```
if dep and dep[0] in (".", "/", "~"):
  # We have something that looks like a path
```

User would have to state `./foo` instead of just `foo` for a local folder... but hey, it's a convenience notation...